etna/transforms/timestamp/holiday.py

```diff
@@ -60,19 +101,41 @@ def _get_column_name(self) -> str:
             return self.__repr__()
 
     def _fit(self, df: pd.DataFrame) -> "HolidayTransform":
+        """Fit the transform.
+
+        Parameters
+        ----------
+        df:
+            Dataset to fit the transform on.
+
+        Returns
+        -------
+        :
+            The fitted transform instance.
         """
-        Fit HolidayTransform with data from df. Does nothing in this case.
+        return self
+
+    def fit(self, ts: TSDataset):
+        """Fit the transform.
 
         Parameters
         ----------
-        df: pd.DataFrame
-            value series with index column in timestamp format
+        ts:
+            Dataset to fit the transform on.
+
+        Returns
+        -------
+        :
+            The fitted transform instance.
         """
+        super().fit(ts=ts)
+        self.freq = ts.freq
         return self
 
     def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
         """
-        Transform data from df with HolidayTransform and generate a column of holidays flags or its titles.
+        Transform data.
```

Comment:
If you are going to write an extended description it should go after a blank line. I don't think we need a description about days_count here, it doesn't add any useful info.